hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_derivative.py

```diff
@@ -684,7 +681,7 @@ async def _execute_cancel(self, trading_pair: str, client_order_id: str) -> str:
                     self.logger().warning(
                         f"Failed to cancel order {client_order_id}:"
                         f" order not found ({response_code} - {response['ret_msg']})")
-                    self.stop_tracking_order(client_order_id)
+                    self._client_order_tracker.process_order_cancelation_failed(client_order_id)
```

Comment:
No need to add extra logic in the ClientOrderTracker. In this case we need to process something like the following: 
```
order_update: OrderUpdate = OrderUpdate(
    trading_pair=trading_pair,
    update_timestamp=self.current_timestamp,
    new_state=OrderState.FAILED,
    client_order_id=client_order_id,
)
self._client_order_tracker.process_order_update(order_update)
return
```